apps/jetbrains/jetbrains.py

```diff
@@ -148,7 +135,15 @@ def idea_commands(commands):
 class Actions:
     def idea(commands: str):
         """Send a command to Jetbrains product"""
-        idea_commands(commands)
+        command_list = commands.split(",")
+        try:
+            for cmd in command_list:
+                if cmd:
+                    send_idea_command(cmd.strip())
+                    actions.sleep(0.1)
```

Comment:
Doesn't have to be done in this PR but we should really see if this can be removed. It makes the running of multiple actions like `idea_grab()` unnecessarily slower/jankier.

(I ran with it removed for many months, but I've since diverged from community enough that it'd be good for someone else to test too)